x-pack/plugins/security_solution/cypress/integration/detection_rules/indicator_match_rule.spec.ts

```diff
@@ -92,104 +110,337 @@ describe('Detection rules, Indicator Match', () => {
+
+      goToRuleDetails();
+
+      cy.get(RULE_NAME_HEADER).should('have.text', `${newThreatIndicatorRule.name}`);
+      cy.get(ABOUT_RULE_DESCRIPTION).should('have.text', newThreatIndicatorRule.description);
+      cy.get(ABOUT_DETAILS).within(() => {
+        getDetails(SEVERITY_DETAILS).should('have.text', newThreatIndicatorRule.severity);
+        getDetails(RISK_SCORE_DETAILS).should('have.text', newThreatIndicatorRule.riskScore);
+        getDetails(REFERENCE_URLS_DETAILS).should((details) => {
+          expect(removeExternalLinkText(details.text())).equal(expectedUrls);
+        });
+        getDetails(FALSE_POSITIVES_DETAILS).should('have.text', expectedFalsePositives);
+        getDetails(MITRE_ATTACK_DETAILS).should((mitre) => {
+          expect(removeExternalLinkText(mitre.text())).equal(expectedMitre);
+        });
+        getDetails(TAGS_DETAILS).should('have.text', expectedTags);
+      });
+      cy.get(INVESTIGATION_NOTES_TOGGLE).click({ force: true });
+      cy.get(ABOUT_INVESTIGATION_NOTES).should('have.text', INVESTIGATION_NOTES_MARKDOWN);
+
+      cy.get(DEFINITION_DETAILS).within(() => {
+        getDetails(INDEX_PATTERNS_DETAILS).should(
+          'have.text',
+          newThreatIndicatorRule.index!.join('')
+        );
+        getDetails(CUSTOM_QUERY_DETAILS).should('have.text', '*:*');
+        getDetails(RULE_TYPE_DETAILS).should('have.text', 'Indicator Match');
+        getDetails(TIMELINE_TEMPLATE_DETAILS).should('have.text', 'None');
+        getDetails(INDICATOR_INDEX_PATTERNS).should(
+          'have.text',
+          newThreatIndicatorRule.indicatorIndexPattern.join('')
+        );
+        getDetails(INDICATOR_MAPPING).should(
+          'have.text',
+          `${newThreatIndicatorRule.indicatorMapping} MATCHES ${newThreatIndicatorRule.indicatorIndexField}`
+        );
+        getDetails(INDICATOR_INDEX_QUERY).should('have.text', '*:*');
+      });
+
+      cy.get(SCHEDULE_DETAILS).within(() => {
+        getDetails(RUNS_EVERY_DETAILS).should(
+          'have.text',
+          `${newThreatIndicatorRule.runsEvery.interval}${newThreatIndicatorRule.runsEvery.type}`
+        );
+        getDetails(ADDITIONAL_LOOK_BACK_DETAILS).should(
+          'have.text',
+          `${newThreatIndicatorRule.lookBack.interval}${newThreatIndicatorRule.lookBack.type}`
+        );
+      });
+
+      waitForTheRuleToBeExecuted();
+      waitForAlertsToPopulate();
+
+      cy.get(NUMBER_OF_ALERTS).should('have.text', expectedNumberOfAlerts);
+      cy.get(ALERT_RULE_NAME).first().should('have.text', newThreatIndicatorRule.name);
+      cy.get(ALERT_RULE_VERSION).first().should('have.text', '1');
+      cy.get(ALERT_RULE_METHOD).first().should('have.text', 'threat_match');
+      cy.get(ALERT_RULE_SEVERITY)
+        .first()
+        .should('have.text', newThreatIndicatorRule.severity.toLowerCase());
+      cy.get(ALERT_RULE_RISK_SCORE).first().should('have.text', newThreatIndicatorRule.riskScore);
+    });
```

Comment:
Don't forget to press this button if you're reviewing this!

![press_this_button](https://user-images.githubusercontent.com/1151048/105904438-5d710b00-5fde-11eb-9ebb-711db2569610.gif)